tests/e2e/tests/critical-path/triggers-and-functions/libraries.e2e.ts

```diff
@@ -41,3 +41,61 @@ test
         await t.expect(row.totalFunctions).eql(item.totalFunctions, 'user name is unexpected');
     });
 
+test.only
+    .after(async() => {
+        await browserPage.Cli.sendCommandInCli(`TFUNCTION DELETE ${libraryName}`);
+        await deleteStandaloneDatabaseApi(ossStandaloneRedisGears);
+    })('Verify that library details is displayed', async t => {
```

Comment:
do you have these tests names into checklist?

Reply:
not yet, I usually updated it after merge and will ask Roma and Amir mark their unit tests